firefox-ios/Client/Frontend/Fakespot/FakespotUtils.swift

```diff
@@ -27,7 +27,7 @@ public struct FakespotUtils: FeatureFlaggable {
 
     public static var termsOfUseUrl: URL? {
         // Returns the predefined URL associated to terms of use button action.
-        return URL(string: "https://www.fakespot.com/privacy-policy?utm_source=review-checker&utm_campaign=privacy-policy&utm_medium=in-product&utm_term=opt-in-screen")
+        return URL(string: "https://www.fakespot.com/terms")
```

Comment:
I'm not 100% clear on whether we still want to include the tracking params (`utm_source` etc.) with this URL change. Can anyone possibly confirm? (cc @DianaBarladeanu @thatswinnie)

Afaics the ticket only mentions changing the destination page (`privacy_policy` -> `terms`) but doesn't specify removing the tracking parameters.

Reply:
@mattreaganmozilla Thank you for your feedback, I added back the tracking params for the URL and I ultimately changed just the destination page. Thanks again 🙏